server/sonar-server/src/main/java/org/sonar/server/measure/index/ProjectMeasuresIndexer.java

```diff
@@ -94,6 +94,16 @@ public void deleteProject(String uuid) {
       .get();
   }
 
+  @Override
+  public void createEsQueueForIndexing(DbSession dbSession, String projectUuid) {
+    // FIXME
+  }
+
+  @Override
+  public void createEsQueueForDeletion(DbSession dbSession, String projectUuid) {
+    // FIXME
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)